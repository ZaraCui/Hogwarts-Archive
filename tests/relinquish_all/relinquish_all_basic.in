ADD COLLECTION tests/data/books_small.csv
ADD STUDENT Harry Potter
RENT 100000 1001
RENT 100000 1002
RELINQUISH ALL 100000
STUDENT SPELLBOOKS 100000
EXIT

