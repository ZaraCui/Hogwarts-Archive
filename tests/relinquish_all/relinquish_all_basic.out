user: 12 spellbooks successfully added.

user: Success.

user: Success.

user: Success.

user: Success.

user: Student not currently renting.

user: Ending Archive process.
